contrib/localnet/scripts/start-upgradehost.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
`start-upgradehost.sh` do we name it like this because the name of the service? I think it would be better to use dash case entirely `start-upgrade-host.sh`

Reply:
added dash to script name and docker service name